client/web/compose/src/components/Public/Record/Modal.vue

```diff
@@ -126,10 +128,11 @@ export default {
       clearRecordIDs: 'ui/clearRecordIDs',
     }),
 
-    loadRecord ({ recordID, recordPageID, values, refRecord }) {
+    loadRecord ({ recordID, recordPageID, values, refRecord, editMode }) {
```

Comment:
i would call all of these just edit, since it will make sense from the context what it is, same for this.modalInEditModal, can be called `edit`